selenium-system-test/src/test/scala/hts/steps/VerifyLinksSteps.scala

```diff
@@ -0,0 +1,108 @@
+
+import scala.collection.JavaConverters._
+
+class VerifyLinksSteps extends Steps {
+
+  Given("""^that users are authenticated$""") { () ⇒
+    AuthorityWizardPage.authenticateUser(AboutPage.expectedURL, 200, "Strong", ScenarioContext.generateEligibleNINO())
+  }
+
+  When("""^they are at the start of the hts pages""") { () ⇒
+    Browser.checkCurrentPageIs(AboutPage)
+  }
+
+  Then("""^they see all feedback, get-help and privacy links are working as they go through the journey$""") { () ⇒
+    Browser.checkCurrentPageIs(AboutPage)
+    checkForLinksThatExistOnEveryPage(AboutPage)
+    Browser.nextPage()
+
+    Browser.checkCurrentPageIs(EligibilityInfoPage)
+    checkForLinksThatExistOnEveryPage(EligibilityInfoPage)
+    Browser.nextPage()
+
+    Browser.checkCurrentPageIs(HowTheAccountWorksPage)
+    checkForLinksThatExistOnEveryPage(HowTheAccountWorksPage)
+    Browser.nextPage()
+
+    Browser.checkCurrentPageIs(HowWeCalculateBonusesPage)
+    checkForLinksThatExistOnEveryPage(HowWeCalculateBonusesPage)
+    Browser.nextPage()
+
+    Browser.checkCurrentPageIs(ApplyPage)
+    checkForLinksThatExistOnEveryPage(ApplyPage)
+
+    ApplyPage.clickStartNow()
+
+    Browser.checkCurrentPageIs(EligiblePage)
+    checkForLinksThatExistOnEveryPage(EligiblePage)
+
+    EligiblePage.clickConfirmAndContinue()
+    checkForLinksThatExistOnEveryPage(SelectEmailPage)
+
+    //try to change the email and verify links
+    SelectEmailPage.setAndVerifyNewEmail("newemail@mail.com")
+    Browser.checkCurrentPageIs(VerifyYourEmailPage)
+    checkForLinksThatExistOnEveryPage(SelectEmailPage)
+
+    //go back to original select email page and continue
+    SelectEmailPage.selectGGEmail()
+    checkForLinksThatExistOnEveryPage(CreateAccountPage)
+
+  }
+
+  private def checkForLinksThatExistOnEveryPage(currentPage: Page): Unit = {
+
+    verifyFeedbackLink()
+    driver.navigate().back()
+    verifyGetHelpLink()
+    verifyPrivacyPolicyLink()
+
+    currentPage.navigate()
+    Browser.checkCurrentPageIs(currentPage)
+  }
+
+  private def verifyFeedbackLink(): Unit = {
+    Browser.isTextOnPage("BETA") shouldBe true
+    Browser.clickButtonByIdOnceClickable("feedback-link")
+    Browser.checkCurrentPageIs(FeedbackPage)
+  }
+
+  private def verifyGetHelpLink(): Unit = {
+    Browser.isTextOnPage("Is there anything wrong with this page") shouldBe true
+    Browser.clickButtonByIdOnceClickable("get-help-action")
+    Browser.isElementByIdVisible("report-error-partial-form") shouldBe true
+
+  }
+
+  private def verifyPrivacyPolicyLink(): Unit = {
+    Browser.isTextOnPage("Privacy policy") shouldBe true
+    Browser.clickLinkTextOnceClickable("Privacy policy")
+    val tabs = driver.getWindowHandles.asScala.toList
+    driver.switchTo.window(tabs(1)) //switch to privacy policy tab
+    Browser.checkCurrentPageIs(PrivacyPolicyPage)
+    driver.switchTo.window(tabs.headOption.getOrElse("missing url here")) //switch back to original page
```

Comment:
since you're doing `tabs(1)` above you might as well do `tabs(0)` here - or you could even do something like:
```scala
tabs match {
  case tab1 :: tab2 :: Nil => // blah blah blah
  case ts                  => fail(s"Unexpected number of tabs: ${ts.getLength}")
}
```